### DISCUSSION TITLE
Differences between Qt5 and Qt6 frontends

### DISCUSSION THREAD

**Discussion by Lepestronik** [CelestiaProject/Celestia#1708] [Q&A] [answered]
It would be interesting to know what are the differences between them and are there any advantages of one over the other, including in the context of different operating systems.

**Comment by 375gnu** [comment, marked as answer]
No differences at all, even the bugs that annoy us in Qt5 are not fixed in Qt6. Qt5 will be superseded by Qt6 one day (actually the next Friday is the last day for Qt5.15 LTS version, so Qt5.15 will receive only community updates).
